packages/toucan-js/src/integrations/zod/zoderrors.ts

```diff
@@ -138,38 +145,39 @@ export function formatIssueMessage(zodError: ZodError): string {
 }
 
 /**
- * Applies ZodError issues to an event context and replaces the error message
+ * Applies ZodError issues to an event extra and replaces the error message
  */
-function applyZodErrorsToEvent(
+export function applyZodErrorsToEvent(
   limit: number,
   event: Event,
-  saveAttachments?: boolean,
-  hint?: EventHint,
+  saveAttachments: boolean = false,
```

Comment:
The only thing I didn't bring in from upstream was renaming this option to avoid breaking anyone who might have used this